versioned_docs/version-6.x.x/core-concepts/validations-and-constraints.md

```diff
@@ -51,11 +51,13 @@ Our code example above defines a unique constraint on the `username` field:
 } /* ... */
 ```
 
-When this model is synchronized (by calling `sequelize.sync` for example), the `username` field will be created in the table as `` `username` TEXT UNIQUE``, and an attempt to insert an username that already exists there will throw a `SequelizeUniqueConstraintError`.
+When this model is [synchronized](https://sequelize.org/docs/v6/core-concepts/model-basics/#model-synchronization), the `username` field will be created in the table as `` `username` TEXT UNIQUE``. An attempt to insert a username that already exists will throw a `SequelizeUniqueConstraintError`.

```

Comment:
```suggestion
When this model is [synchronized](https://sequelize.org/docs/v6/core-concepts/model-basics/#model-synchronization) (by calling `sequelize.sync`), the `username` field will be created in the table as `` `username` TEXT UNIQUE``. An attempt to insert a username that already exists will throw a `SequelizeUniqueConstraintError`.
```

And, how to link to an internal doc correctly?

Reply:
You can use a relative link to the markdown file, here is an example: https://github.com/sequelize/website/blob/6fa8e600d7d3cc83dc628857bb6dbb4b1719e2d3/docs/index.mdx?plain=1#L84